tests/integration/smb_share_test.go

```diff
@@ -203,7 +203,7 @@ func (s *SmbShareWithDNSSuite) TestShareAccessByDomainName() {
 	// test that the IP in ad dns matches the service
 	ctx, cancel := context.WithDeadline(
 		context.TODO(),
-		time.Now().Add(30*time.Second))
+		time.Now().Add(60*time.Second))
```

Comment:
All the changes look good and I have only one small request. 
Instead of using a number here directly, can you use a variable defined as a global in tests/integration/util_test.go. This is far more easily maintainable.

Reply:
These changed are now included with PR #162, thanks.